src/lib/import-export/import/importers/importer.ts

```diff
@@ -114,6 +115,7 @@ abstract class BaseBackupImporter extends BaseImporter {
 				wpContentDirectory: this.backup.wpContentDirectory,
 				wpConfig: this.backup.wpConfig,
 				meta: this.meta,
+				importerType: this.constructor.name,
```

Comment:
```suggestion
				importerType: StatsMetric.SQL_IMPORTER,
```
Defining the value by referencing the enum directly seems more understandable.

Reply:
I find it strange to reference stats props in the importer class.
What do you think about using a similar approach to the one we implemented for `getPlatform` ( check `getImporterMetric` )?

Reply:
I like the new approach 👍